docs/guide-end-of-form.rst

```diff
@@ -185,23 +205,41 @@ In many workflows, it's important to guarantee that data is not changed after th
 
 When data collectors reach the form end screen, they only have the option to :guilabel:`Finalize`. If they are interrupted during a form filling session, they need to exit and discard changes or rely on automatic data backups and recovery (the partially-filled form will open automatically when they open the same blank form again).
 
-Edits may be needed to some drafts
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-In some workflows, new information may need to be added to a form after a first data collection event. For example:
+.. note::
+  If you would still like to leave the door open to edits but they are generally strongly discouraged, you could leave the option to save as draft from the form filling experience and use :ref:`audit logging <guide-form-states-form-audit-log>` with change tracking and reasons for change to get more information on edits that are made.
```

Comment:
s/leave the door open to/allow for/

You can link directly to change tracking now.